src/app/inscriptions.ts

```diff
@@ -117,6 +118,24 @@ const Inscriptions = function (config) {
             existingInscription.value > newInscription.value ||
             (existingInscription.value === newInscription.value &&
                 existingInscription.created_at < newInscription.created_at),
+        isSpent: async (utxo) => {
```

Comment:
can you move this to the utxo.js file instead? It seems a more appropiate place

Reply:
done @andredevjs